README.md

```diff
@@ -18,20 +18,25 @@ Allows to create a gulp files in classes, each method of which can be a gulp tas
     
     ```typescript
     import {Gulpclass, Task, SequenceTask} from "gulpclass";
-
-    let gulp = require("gulp");
-    let del = require("del");
+    import * as gulp from "gulp";
+    import * as del from "del";

```

Comment:
Typings are integral part of TypeScript ecosystem, we shouldn't try to hide this or to protect new users from it. I think, the sooner they will encounter and learn this subject — the better for them.